data/json/professions.json

```diff
@@ -4948,6 +5321,43 @@
       "female": { "entries": [ { "item": "bra" }, { "item": "panties" } ] }
     }
   },
+  {
+    "type": "profession",
+    "id": "lobster",
+    "name": { "male": "Lobsterman", "female": "Lobsterwoman" },
+    "description": "Out on the briny deep, you've spent your days haulin' traps, ridin' the swells, and huntin' them elusive crustaceans.  A line of work not suitable for the faint of heart.  At least the lobsters didn't got too big to seek vengeance - Yet.",
+    "points": 2,
+    "skills": [ { "level": 5, "name": "survival" }, { "level": 2, "name": "swimming" } ],
+    "proficiencies": [ "prof_fibers", "prof_knives_familiar" ],
+    "pets": [ { "name": "mon_fish_lobster", "amount": 1 } ],
+    "items": {
+      "both": {
+        "entries": [
+          { "item": "hat_cotton" },
+          { "item": "jeans" },
+          { "item": "tshirt", "variant": "generic_tshirt" },
+          { "Item": "gloves_rubber" },
+          { "item": "socks" },
+          { "item": "fishing_waders" },
+          { "item": "wristwatch" },
+          { "item": "backpack" },
+          { "Item": "pliers" },
+          { "item": "lobster_cage", "custom-flags": [ "auto_wield" ] },
+          { "Item": "lobster_smoked", "container-item": "wrapper" },
+          { "Item": "lobster_smoked", "container-item": "wrapper" },
+          { "Item": "lobster_smoked", "container-item": "wrapper" },
+          { "group": "charged_smart_phone" },
+          { "item": "fish_trap", "ammo-item": "fish_bait", "charges": 5 },
+          { "item": "fish_bait", "charges": 15 },
+          { "item": "lighter", "charges": 100 },
+          { "item": "knife_hunting", "container-item": "sheath" },
+          { "item": "net", "custom-flags": [ "auto_wield" ] }
+        ]
+      },
+      "male": { "entries": [ { "item": "boxer_shorts" } ] },
+      "female": { "entries": [ { "item": "bra" }, { "item": "panties" } ] }
+    }
+  },
```

Comment:
This is just silly, fishermen aren't walking around hauling lobster pots and chunks of smoked meat.
The survival number is also bizzarely high?
```suggestion
```